manticore/native/memory.py

```diff
@@ -278,7 +278,7 @@ def __init__(self, start, size, perms, data_init=None, name=None, **kwargs):
         :param perms: the access permissions of the map.
         :param data_init: the data to initialize the map.
         """
-        super().__init__(start, size, perms, name, **kwargs)
+        super().__init__(start, size, perms, name)
```

Comment:
Note, previously, the `**kwargs` passed here would have never worked properly if there were any key/value pairs there.  mypy revealed this problem after adding some type hints to this constructor.

Reply:
Good catch!

Reply:
Also, it looks like there is no docstring for `name`, which was probably added at a different time than the other arguments, and `name` was probably meant to be caught in the `**kwargs`.

Not sure what we actually want to do to resolve this. I'm not a huge fan of using `kwargs` here, since we know all of the arguments of `Map`. As long as we're consistent in the order the arguments are written in the function signature, then we should list them all explicitly. However, we would also have be copying the parameter docstring(s) as well, which can desynchronize easily :/ . Maybe there is a way to link to `Map`'s docstring for certain parameters?